package.json

```diff
@@ -107,12 +104,9 @@
     "node": "^10 || ^12 || ^14"
   },
   "dependencies": {
-    "@plone/volto": "13.8.2",
```

Comment:
From pulling and testing against the version I was running on my desktop, the critical difference _seems_ to be setting the volto-plotlycharts dependency to a version number for yarn to handle rather than the Github repo.